### DISCUSSION TITLE
Hyperparameters for the trained models

### DISCUSSION THREAD

**Discussion by AlaaKhaddaj** [mlfoundations/open_clip#350] [Q&A] [answered]
Hello, 

Do you have a list of **all** the hyperparameters used for training your models? (e.g. lr, batch size, grad ckpt, grad accum, etc.) 
I am am particularly interested in the models trained on CC12M and LAION400M.  

Thanks!

**Comment by mitchellnw** [comment, marked as answer]
If you check out here, for instance, you can see the slrum script used to train the model with the hparams: https://huggingface.co/laion/CLIP-ViT-L-14-laion2B-s32B-b82K
